graphql_compiler/tests/test_data_tools/data/commands.sql

```diff
@@ -11,238 +11,878 @@ create vertex Animal set birthday = DATE("2006-09-20 00:00:00"), color = 'orange
+create vertex Animal set birthday = DATE("2005-05-16 00:00:00"), color = 'black', name = 'Pteranodon__(((1013155)131719)0118)', net_worth = 732.31, uuid = '42965873-39d5-b552-a867-a0f0a8e6a772'
+create edge Animal_ParentOf from (select from Animal where name = 'Pteranodon__(((1013155)131719)0118)') to (select from Animal where name = 'Pteranodon__((1013155)131719)')
+create edge Animal_ParentOf from (select from Animal where name = 'Pteranodon__(((1013155)131719)0118)') to (select from Animal where name = 'Pteranodon__0')
+create edge Animal_ParentOf from (select from Animal where name = 'Pteranodon__(((1013155)131719)0118)') to (select from Animal where name = 'Pteranodon__1')
+create edge Animal_ParentOf from (select from Animal where name = 'Pteranodon__(((1013155)131719)0118)') to (select from Animal where name = 'Pteranodon__18')
+create vertex Animal set birthday = DATE("2006-08-04 00:00:00"), color = 'orange', name = 'Pteranodon__(01163)', net_worth = 921.98, uuid = '53a081a6-a764-7989-f20e-8bda39cc6d88'
+create edge Animal_ParentOf from (select from Animal where name = 'Pteranodon__(01163)') to (select from Animal where name = 'Pteranodon__0')
+create edge Animal_ParentOf from (select from Animal where name = 'Pteranodon__(01163)') to (select from Animal where name = 'Pteranodon__1')
+create edge Animal_ParentOf from (select from Animal where name = 'Pteranodon__(01163)') to (select from Animal where name = 'Pteranodon__16')
+create edge Animal_ParentOf from (select from Animal where name = 'Pteranodon__(01163)') to (select from Animal where name = 'Pteranodon__3')
+create vertex Animal set birthday = DATE("2007-04-15 00:00:00"), color = 'yellow', name = 'Pteranodon__(121738)', net_worth = 735.63, uuid = 'bf4fa4a3-c908-53fd-fc9e-a692ba626aee'
+create edge Animal_ParentOf from (select from Animal where name = 'Pteranodon__(121738)') to (select from Animal where name = 'Pteranodon__12')
+create edge Animal_ParentOf from (select from Animal where name = 'Pteranodon__(121738)') to (select from Animal where name = 'Pteranodon__17')
+create edge Animal_ParentOf from (select from Animal where name = 'Pteranodon__(121738)') to (select from Animal where name = 'Pteranodon__3')
+create edge Animal_ParentOf from (select from Animal where name = 'Pteranodon__(121738)') to (select from Animal where name = 'Pteranodon__8')
+create vertex Animal set birthday = DATE("2011-08-24 00:00:00"), color = 'magenta', name = 'Pteranodon__((((1013155)131719)0118)11219)', net_worth = 699.6, uuid = '2bdb5720-63ea-a07f-f436-498e68afe2b8'
+create edge Animal_ParentOf from (select from Animal where name = 'Pteranodon__((((1013155)131719)0118)11219)') to (select from Animal where name = 'Pteranodon__(((1013155)131719)0118)')
+create edge Animal_ParentOf from (select from Animal where name = 'Pteranodon__((((1013155)131719)0118)11219)
```

Comment:
If we are already storing the provenance of the animals, thoughts on making the names not be ambiguous? This can be parsed in several ways: `<stuff>, 13, 15, 2`, `<stuff>, 13, 1, 52` etc.